pkgs/development/coq-modules/coq-extensible-records/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchurl, coq }:
+
+stdenv.mkDerivation {
+  name = "coq${coq.coq-version}-coq-extensible-records-1.2.0";
+
+  src = fetchurl {
+    url = https://github.com/gmalecha/coq-extensible-records/archive/1.2.0.tar.gz;
```

Comment:
Please use `fetchzip` or `fetchFromGitHub` (in case this tarball is regenerated, the meta-data of the files it holds may change, which is enough to invalidate the hash computed by `fetchurl`).

Reply:
Switched to `fetchFromGitHub`, sorry!

Reply:
You can use the package version as value of the `rev` field. It makes the description less obscure and updating easier (if the value of the version is written only once).